js/main.js

```diff
@@ -80,6 +80,8 @@ function toggleHighlighting(player) {
   }
 }
 
+// todo --> I've tried abstracting these class changes out, but I can't get it to work. I'd appreciate feedback about how
+//  I could do that if possible, so it can be a dynamic function instead of being so rigid
 function checkEmptyDeck(player) {
```

Comment:
There might be a way, I would want to play with it on your code, so maybe we can work on it together later. My thought is that player is always going to be player1 or player2, which is how you set up all you variables you are changing. Maybe you can concatenate player with the classList add/remove to be truly dynamic?